vmcnet/train/runners.py

```diff
@@ -618,7 +618,11 @@ def run_molecule() -> None:
         and reload_config.use_checkpoint_file
     )
 
-    if reload_from_checkpoint:
+    if config.reload_model_state!="":
+        params=utils.io.reload_model_state(config.reload_model_state)
+        params=jax.tree_map(lambda x: x[None,...], params)
```

Comment:
For consistency you can use `params = utils.io.replicate_all_local_devices(params)` here.

Also, it looks like the preexisting reload_from_checkpoint logic below assumes the data needs to be pmapped and doesn't handle the case where config.distribute=False. Could you make the broadcasting/distributing logic dependent on config.distribute=True while you're in here?